operations/workplace/README.md

```diff
@@ -4,31 +4,31 @@ description: Description of norms on Mattermost Community server
 
 # Workplace
 
-## What is the Mattermost pre-release server?
+## What is the Mattermost Community Server?
 
 The Mattermost pre-release server runs the latest builds of Mattermost's software, including all open source code for Mattermost Team Edition as well as commercial code for Mattermost Enterprise Edition.
 
 It has three objectives:
 
-* Provide a workplace for Mattermost Staff 
-* Test pre-released builds of our core products
-* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners 
+* Provide a workplace for Mattermost staff.
+* Test work-in-progress builds of our core products.
+* Engage with our [communities ](../../contributors/contributors/community.md)including contributors and partners.
 
-**The Mattermost pre-release server is NOT A PRODUCTION SERVER.** It is a live view of work-in-progress and will contain bugs and have occasional downtime.
+**The Mattermost Community server is NOT A PRODUCTION SERVER.** It is a live view of work-in-progress and will contain bugs and have occasional downtime.
 
 You can reach the server at:
 
-{% embed url="https://pre-release.mattermost.com" caption="" %}
+{% embed url="https://community-release.mattermost.com" caption="" %}
 
-The server is also available via the URL [https://community.mattermost.com](https://community.mattermost.com) which receives the same regular pushes of partially tested code as [https://pre-release.mattermost.com](https://pre-release.mattermost.com) and [https://daily-community.mattermost.com](https://daily-community.mattermost.com) which offers daily builds.
+The server is also available via the URL [https://community-release.mattermost.com](https://community-release.mattermost.com). This server receives regular pushes of features and/or bug fixes of the next release for testing purposes. Additionally, you can use [https://community-daily.mattermost.com](https://community-daily.mattermost.com) which offers daily builds. 
 
 The server has two teams:
 
 ### **Contributors Team**
 
 URL: [https://community.mattermost.com/core](https://community.mattermost.com/core/messages/@github)
 
-Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://pre-release.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator
+Available to anyone who'd like to join the server using either an email and password account \([you can sign-up for a free account online](https://community.mattermost.com/signup_user_complete)\) or a Mattermost staff account, which is set up via OneLogin and requires MFA either through OneLogin or Google Authenticator.
 
 ### **Staff Team**
```

Comment:
There's a formatting error at the end of the second sentence here (`... MFA**.**`)

![image](https://user-images.githubusercontent.com/13119842/78721537-beb3da80-78f5-11ea-9352-3e2b22b055d4.png)